### PR TITLE
Configure watch/livereload extension and document it.

### PR DESCRIPTION
**Pull request by Carreau**
Is usefull mostly for css, but will be for typscript too.

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -52,8 +52,8 @@
 
 # General information about the project.
 project = 'Jupyter Notebook'
-copyright = '2015, https://er'
-author = 'https://er'
+copyright = '2015, Jupyter/IPython Team, https://jupyter.org'
+author = 'The Jupyter/IPython Team'
```
Comment (minrk) [inline docs/source/conf.py]: just Jupyter

File: docs/source/development.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+If you are working on notebook styling, you can use the live-reloading option
+of gulp to watch for changes in the `.less` files and automatically rebuild the
+`css` and reload the current page. 
+
+To do so we rely on the livereaload extension of various browser (`for chrome
+<https://chrome.google.com/webstore/detail/livereload/jnihajbhpnppcggbcgedagnkighmdlei>`_),
+and the gulp-livereload extension. 
+
+
+To use the livereload extension, in the root of Jupyter_notebook project launch::
```
Comment (minrk) [inline docs/source/development.rst]: s/J/j

**Comment by minrk** [comment]
@Carreau needs rebase.

**Comment by Carreau** [comment]
Painful rebase (just kidding).

**Pull request merged**